language/move-binary-format/src/proptest_types.rs

```diff
@@ -348,9 +351,10 @@ impl CompiledModuleStrategyGen {
                         identifiers,
                         address_identifiers,
                         constant_pool,
-                    }
-                    .freeze()
-                    .expect("valid modules should satisfy the bounds checker")
+                    };
+                    BoundsChecker::verify_module(&module)
```

Comment:
Same as the other, you could probably remove these if you wanted. Should be verifying elsewhere